docs/tutorials/multimodal/beginner_image_cls.md

```diff
@@ -0,0 +1,88 @@
+# AutoMM for Image Classification: Quick Start
```

Comment:
Need to unify the title format to use either `:` or `-`. Other tutorials seem to use `-`.

Reply:
Just changed